.github/workflows/deploy-website.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
 
       - name: Checkout Repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v1
```

Comment:
Why the change to v1?

Reply:
v2 does only a shallow (1 level deep checkout), so it can't be used to determine if files changed for specific rebuilds if that happened more than one commit ago.